frontend/simple/components/Proposals/Proposals.vue

```diff
@@ -0,0 +1,173 @@
+      :proposal="proposal"
+    />
+    <!--  End Original Voting Banner to delete soon -->
+
+    <h4 class="title is-size-5 notification is-warning gi-is-banner gi-notify"
+      v-if="groupProposals.notVoted.length"
+    >
+      <i18n>These are waiting for your vote!</i18n>
+    </h4>
+    <voting
+      v-for="proposal in groupProposals.notVoted"
+      :proposal="proposal"
+    />
+
+    <h4 class="title is-size-5" v-if="groupProposals.own.length">
+      <i18n>Your own proposals</i18n>
+    </h4>
+    <voting
+      v-for="proposal in groupProposals.own"
+      :proposal="proposal"
+    />
+
+    <a class="gi-showOther has-text-weight-bold"
+      v-if="!showOtherProposals && groupProposals.alreadyVoted.length"
+      @click.prevent="showOtherProposals = true"
+    >
+      <i18n>Show {{groupProposals.alreadyVoted.length}} voted proposals</i18n>
+    </a>
+
+    <h4 class="title is-size-5" v-if="showOtherProposals">
+      <i18n>Voted proposals</i18n>
+    </h4>
+
+    <voting
+      v-if="showOtherProposals"
+      v-for="proposal in groupProposals.alreadyVoted"
+      :proposal="proposal"
+    />
+  </dashboard-section>
+</template>
+<style lang="scss" scoped>
+@import "../../sass/theme/index";
+
+.gi-notify {
+  margin-top: $gi-spacer-lg;
+  margin-bottom: -$gi-spacer;
+}
+
+.gi-showOther {
+  display: block;
+}
+</style>
+<script>
+import DashboardSection from '../DashboardSection.vue'
+import Voting from './Voting.vue'
+import VotingBanner from '../VotingBanner.vue'
+
+export default {
+  name: 'Proposals',
+  components: {
+    DashboardSection,
+    Voting,
+    VotingBanner
+  },
+  props: {
+    proposals: Object
+  },
+  data () {
+    return {
+      showOtherProposals: false,
+      mockProposals: [
+        {
+          type: 'member',
+          member: {
+            picture: 'images/default-avatar.png',
+            name: 'Sam'
+          },
+          title: 'Remove Member',
+          text: 'remove Gil from the group.',
+          ctas: {
+            for: 'Yes, remove',
+            against: 'Keep Sam'
+          },
+          initiator: 'Sam',
+          members: 7,
+          votes: 4,
+          userVote: null
+        },
+        {
+          type: 'rule',
+          value: 0.6,
+          title: 'Rule - Add Memmber',
+          text: 'change approval of Add Member from 80% to 60%',
+          ctas: {
+            for: 'Yes, change',
+            against: 'Keep 80%'
+          },
+          initiator: 'Karl',
+          members: 7,
+          votes: 0,
+          userVote: null
+        },
+        {
+          type: 'rule',
+          value: 0.92,
+          title: 'Rule - Change Mincome',
+          text: 'change approval of Change Mincome from 87% to 92%',
+          ctas: {
+            for: 'Yes, change',
+            against: 'Keep 87%'
+          },
+          initiator: 'Karl',
+          members: 7,
+          votes: 1,
+          userVote: false
```

Comment:
@dotmacro 
> Regarding the prop proposal:
> Because Sam is bold and Gil is not, it would be easy to misread that the proposal is to "Remove Member Sam" 

I fully agree with you, you can see [on the mockups](https://user-images.githubusercontent.com/14869087/39841810-f5ab1c74-53db-11e8-8bff-75e40315d24a.png) that is exactly how you said.

Design:
- **Sam** has proposed **change approval of Add Member** from 80% to **60%**

And still I'm not sure if it's the best option. Maybe it could be:

- [Sam]() has proposed **change Add Member Rule from 80% to 60%**
- [Sam]() has proposed **invite [Kim]() to the group**

> The proposal is to remove Gil, but the vote button says "Keep Sam" (as in ctas)

Yes, that was a typo! (ups) It should say "Keep Gil".  
BTW @taoeffect, **CTA** stands for **Call to Action**. Is very used on product/marketing vocabulary. It's the main action the user should perform on a particular task.

And actually looking at current CTAs a few days later since I've design/implement it...
- 'Yes, remove',
- 'Keep Gil'

I think they are not consistent. It should be:

- 'Yes, Remove Gil',
- 'No, Keep Gil'

or only

- 'Remove Gil',
- 'Keep Gil'

This little change might seem not relevant, but it's proven how micro copywriting have a big impact on the user cognitive process to understand exactly what needs to be done.

Reply:
> you can see on the mockups that is exactly how you said.

Yes, the mockups look good 👍 

> looking at current CTAs a few days later .... I think they are not consistent
> micro copywriting have a big impact on the user cognitive process

Both of your consistent suggestions (with or without yes/no) seem unbiased and clear.